plans/models.py

```diff
@@ -429,6 +444,16 @@ class Order(models.Model):
     created = models.DateTimeField(_('created'), db_index=True)
     completed = models.DateTimeField(
         _('completed'), null=True, blank=True, db_index=True)
+    plan_extended_from = models.DateField(
+        _('The plan was extended from this date'),
+        null=True,
+        blank=True,
+    )
+    plan_extended_until = models.DateField(
+        _('The plan was extended until this date'),
```

Comment:
This is not the right way of using `verbose_name`, this is more suited for `help_text`.

Reply:
Fixed in #77